src/Admin/BlockAdmin.php

```diff
@@ -233,4 +202,69 @@ private function getDefaultTemplate(BlockServiceInterface $blockService)
             return $options['template'];
         }
     }
+
+    private function configureBlockFields(FormMapper $formMapper, BlockInterface $block): void
+    {
+        $service = $this->blockManager->get($block);
+        $blockType = $block->getType();
```

Comment:
```suggestion
```

Reply:
Why? We need this in line 214

Reply:
`$blockType` is not currently used at line 214, since `$block->getType()` is being called again. If that line is reached, the execution flow breaks there since it is used inside a thrown exception.